pkg/cli/admin/node/logs.go

```diff
@@ -273,15 +273,21 @@ func (o LogsOptions) RunLogs() error {
 			}
 			if len(o.Units) > 0 {
 				for _, unit := range o.Units {
+					// Needed to allow working with kubelet that does not support query
 					req.Param("unit", unit)
+					req.Param("query", unit)
```

Comment:
Shouldn't we have a some kind of transformation instead directly assigning?

Reply:
No transformation is required as such.